.github/workflows/scc-linux.yml

```diff
@@ -23,7 +23,7 @@ jobs:
           python-version: '3.13'
 
       - name: Install dependencies
-        run: pip install build setuptools ioctl-opt libusb1 toml vdf pytest
+        run: pip install build setuptools ioctl-opt libusb1 tomli vdf pytest
```

Comment:
Installing tomli here should not be necessary as it is in core 3.11 but the CI uses 3.13 already

Reply:
Okay, but the internal version is called tomllib, so I'd need to change the Python code too. I guess that's fine even if you want to support older versions because it's just for the tests.

Reply:
Sounds good, let's just use the core version for tests, I don't see toml being used anywhere else in this codebase anyway